server/routes/addChainNode.ts

```diff
@@ -23,6 +24,13 @@ const addChainNode = async (models, req: Request, res: Response, next: NextFunct
   if (!req.body.id || !req.body.name || !req.body.symbol || !req.body.network || !req.body.node_url || !req.body.base) {
     return next(new Error(Errors.MissingParams));
   }
+  if (req.body.substrate_spec) {
```

Comment:
We should ideally do server-side validation of the spec, using the same logic from `/editSubstrateSpec`, and reject the chain if it doesn't work.